src/telemetry/TelemetryEvent.ts

```diff
@@ -1,7 +1,11 @@
 enum TelemetryEvent {
     ValidateOmnichannelConfig = "ValidateOmnichannelConfig",
     ValidateSDKConfig = "ValidateSDKConfig",
-    InitializeChatSDK = "InitializeChatSDK",
+    InitializeChatSDKSequential = "InitializeChatSDKSequential",
```

Comment:
I see that we introduced two new events on initialization: `InitializeChatSDKSequential` & `InitializeChatSDKParallel`. 

Previously, we'd only have `InitializeChatSDK`. Is there any monitors that are dependent on `InitializeChatSDK` events? If monitors or other components dependent on this event would potentially stop working.

In order to keep `back comp`, keeping `InitializeChatSDK` for the sequential flow and using `InitializeChatSDKUsingConcurrency` for the "concurrent" flow may be safer.